pkg/services/user/model.go

```diff
@@ -257,6 +259,24 @@ func (u *SignedInUser) IsRealUser() bool {
 	return u.UserID != 0
 }
 
+func (u *SignedInUser) IsApiKeyUser() bool {
+	return u.ApiKeyID != 0
```

Comment:
```suggestion
	return u.ApiKeyID > 0
```

We've had scenarios where this values was changed to -1 and that does not correspond to a real user or API Key

Reply:
Changed, thanks!